docs/source/en/perf_train_cpu_many_k8s.md

```diff
@@ -0,0 +1,271 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# Efficient Training on Multiple CPUs from a Kubernetes Cluter
+
+Using multiple CPUs accelerates the model training process by distributing the workload across multiple nodes. This
+guide builds upon the [Distributed CPU Training](https://huggingface.co/docs/transformers/perf_train_cpu_many) example
+to move the workload to a [Kubernetes](https://kubernetes.io) cluster.
+
+[Kubeflow](https://www.kubeflow.org) simplifies the process of deploying machine learning workloads to Kubernetes
+clusters. For this example, the [Kubeflow PyTorchJob training operator](https://www.kubeflow.org/docs/components/training/pytorch/)
+is used to deploy the distributed training job to the cluster. For efficient training on CPUs,
+[Intel® Extension for PyTorch](https://github.com/intel/intel-extension-for-pytorch) and
+[Intel® oneCCL Bindings for PyTorch](https://github.com/intel/torch-ccl) libraries are used.
+
+## Cluster setup
+
+This guide assume that you already have a Kubernetes clusters with multiple CPUs that will be used to run the
+distributed training job. Use `kubectl get nodes` to see a list the nodes that are available on your cluster. Before
+running the distributed training job on the cluster, Kubeflow needs to be installed and there needs to be a storage
+location that can be used for the dataset and model files.
+
+### Kubeflow Install
+
+Follow the [Kubeflow installation](https://www.kubeflow.org/docs/started/installing-kubeflow/) guide to deploy the
+Kubeflow resources to your cluster. To verify that the PyTorch custom resource has been deployed to your cluster, use
+`kubectl get crd` and ensure that the output includes `pytorchjob.kubeflow.org` like the sample output below:
```

Comment:
Or maybe reword it like:
use `kubectl get crd pytorchjobs.kubeflow.org` and verify the output is similar to: